### PR TITLE
xcinfo 1.0.2

### PR DESCRIPTION
**Pull request by trispo**
Update to xcinfo 1.0.2

### CODE REVIEW

File: Formula/xcinfo.rb [hunk 1]
```diff
@@ -15,9 +15,8 @@ class Xcinfo < Formula
     sha256 cellar: :any_skip_relocation, catalina:       "4cd605c355e9c18be2219e5b459847673066ada976afee46c49f4b667c91fba5"
   end
 
-  depends_on xcode: ["12.4", :build]
-  depends_on macos: :catalina
-  depends_on :macos
```
Comment (chenrui333) [inline Formula/xcinfo.rb:20]: can you add this back?, and remove line 19 (I believe it would be covered by xcode version constraint)

Comment (trispo) [reply Formula/xcinfo.rb:20]: Well, the build needs Xcode 14.

Comment (SMillerDev) [reply Formula/xcinfo.rb:20]: So then they should be updated, not removed

Comment (trispo) [reply Formula/xcinfo.rb:20]: Do you mean I should re-add
`-  depends_on :macOS`?

Comment (trispo) [reply Formula/xcinfo.rb:20]: Done that

Comment (trispo) [reply Formula/xcinfo.rb:20]: > You should re-add all of them with the correct information

Can you please guide me what the correct information is?

File: Formula/xcinfo.rb [hunk 2]
```diff
@@ -15,9 +15,9 @@ class Xcinfo < Formula
...
+  depends_on xcode: ["14.2", :build]
+  depends_on macos: :ventura
+  depends_on macos:
```
Comment (SMillerDev) [inline Formula/xcinfo.rb:20]:
```suggestion
  depends_on :macos
```

Comment (trispo) [reply Formula/xcinfo.rb]: 🙈

File: Formula/xcinfo.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/xcinfo.rb:20]: Looks fine now based on your earlier comment.

Comment (chenrui333) [reply Formula/xcinfo.rb:20]: Thanks!

**Review by chenrui333 — DISMISSED** [review]

**Pull request closed without merging**